kie-server-parent/kie-server-remote/kie-server-client/src/main/java/org/kie/server/client/impl/UserTaskServicesClientImpl.java

```diff
@@ -828,17 +828,25 @@ public TaskInstance findTaskByWorkItemId(Long workItemId) {
     }
 
     @Override
-    public TaskInstance findTaskById(Long taskId) {
+    public TaskInstance findTaskById(Long taskId){
+        return findTaskById(taskId, false);
+    }
+
+    @Override
+    public TaskInstance findTaskById(Long taskId, boolean withSLA) {
         if( config.isRest() ) {
             Map<String, Object> valuesMap = new HashMap<String, Object>();
             valuesMap.put(TASK_INSTANCE_ID, taskId);
 
+            String queryString = "";
```

Comment:
you probably dont even need this, just send the value of _withSLA_ always?

Reply:
@cristianonicolai sure, updated, Thanks!